package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@eppo/js-client-sdk-common",
-  "version": "1.5.2",
+  "version": "1.6.0",
```

Comment:
Minor version increment. Feels right for the scope of changes to accommodate obfuscation, but am open to change this to a patch increment if that makes more sense

Reply:
Cool - the ultimate version to add to the "capabilities" in the API will be governed by the upstream (nodejs, js, react) projects.

Just a thought: Could be nice to synchronize them as it seems unlikely to be publishing very much to one platform over commons.

Reply:
Yup, the version for "capabilities" will be from the js client and react native SDKs

Hmm auto-synchronizing (if I understand you correctly) might be tricky since we need code changes to get the non-common SDKs up to date. E.g. I'm thinking of overriding the `getAssignment` methods and calling the common version within with the `obfuscated` argument set to `true`

```
export class EppoJSClient extends EppoClient {
  public static instance: EppoJSClient = new EppoJSClient(localStorage);

  getAssignment(
    subjectKey: string,
    flagKey: string,
    subjectAttributes?: Record<string, any>,
    assignmentHooks?: IAssignmentHooks,
  ): string {
    return super.getAssignment(subjectKey, flagKey, subjectAttributes, assignmentHooks, true);
  }

  ...
}
```